packages/main/src/Title.js

```diff
@@ -16,14 +16,21 @@ const metadata = {
 	properties: /** @lends sap.ui.webcomponents.main.Title.prototype */ {
 
 		/**
-		 * Defines whether the component would wrap.
+		 * Determines the wrapping type of the component.
+		 * <br><br>
+		 * The available options are:
+		 * <ul>
+		 * <li><code>None</code> - The title will be truncated with an ellipsis.</li>
+		 * <li><code>Normal</code> - The title will wrap. The words won't break based on hyphenation.</li>
+		 * </ul>
 		 *
-		 * @type {boolean}
-		 * @defaultvalue false
+		 * @type {string}
+		 * @defaultvalue "None"
 		 * @public
-		*/
-		wrap: {
-			type: Boolean,
+		 */
+		wrappingType: {
+			type: String,
```

Comment:
The property should not be of type String, but rather enum, called WrappingType which currently has 2 options: None & Normal.

Reply:
After a quick discussion - we think both Wrapping and WrappingType sound ok, but we preferred WrappingType at the end.

Reply:
Added the new enum.